httpcore/__init__.py

```diff
@@ -57,4 +57,5 @@
 
 for _name in __all__:
     if not _name.startswith("__"):
+        __locals[_name].__full_module__ = __locals[_name].__module__  # Used by Shpinx.
```

Comment:
*Sphinx

Reply:
That's up to date in the latest diff. :)